crates/net/network/src/peers/manager.rs

```diff
@@ -127,7 +129,13 @@ impl PeersManager {
             ban_list,
             ban_duration,
             backoff_duration,
+        };
+
+        for NodeRecord { address, tcp_port, udp_port: _, id } in trusted_nodes {
+            manager.add_discovered_node(id, SocketAddr::from((address, tcp_port)));
         }
+
```

Comment:
again, this will trigger events that we don't want

Reply:
Fixed! Peers are added directly to the hashmap on new, now. Should we add a call to `fill_outbound_slots` after that?